tagger.py.in

```diff
@@ -2,18 +2,20 @@
 
 import os
 import sys
+import multiprocessing
 
 
-sys.path.insert(0, '.')
 
+sys.path.insert(0, '.')
```

Comment:
This change isn't required.

Reply:
My bad, was trying to fix the onefile issue and ended up passing.